README.md

```diff
@@ -45,6 +45,6 @@ Additionally, if the pending root is ready to be accepted but a root updater sug
 
 This can lead to infinite loops of pending root if an automation mechanism suggests a root at an interval smaller than the timelock.
 
-This behavior is acknowledged. When designing a strategy on top of the URD, you must ensure that the epoch interval at which you update the root is longer than the timelock. Define these parameters accordingly.
+This behavior is acknowledged. When designing a strategy on top of the Urd, you must ensure that the epoch interval at which you update the root is longer than the timelock. Define these parameters accordingly.
```

Comment:
Just want to point out that it could be formatted URD here because it's natural language outside of code conventions ahah (but I'm fine with both)